docs/db/week-3/lesson.md

```diff
@@ -80,19 +80,12 @@ In the following, we will add a new API endpoint to create a new hotel in the ta
 INSERT INTO hotels (name, rooms, postcode) VALUES ('New Hotel', 5, 'ABC001');
 ```
 
-As we create a new record in the database, we will add a new POST endpoint in the `cyf-hotels-api` project from last class. Moreover, we need to be able to pass some parameters to this API endpoint such as the hotel name, the number of rooms and the postcode, so we can use this API to create different hotel. These parameters can be sent in the body of the request. To access the parameters in the body of the request with Express.JS, we need to add the module `body-parser` to the `cyf-hotels-api` project:
-
-```
-npm install --save body-parser
-```
-
-Then include it in the `server.js`:
+As we create a new record in the database, we will add a new POST endpoint in the `cyf-hotels-api` project from last class. Moreover, we need to be able to pass some parameters to this API endpoint such as the hotel name, the number of rooms and the postcode, so we can use this API to create a different hotel. These parameters can be sent in the body of the request. To access the parameters in the body of the request with Express.JS, we need add the following line in the `server.js` file of the `cyf-hotels-api` project:
```

Comment:
Should this include a direction about _where_ in the file to include it? e.g. something like "After the define the `app` variable" or "Before we declare the first handler" or "before the call to `app.listen`" or similar?

Reply:
Yes, I wondered about this, but I don't think it matters, as long as it's after app is instantiated, it's just another bit of configuration that is only used once the app.listen is called.
At least that what I believe, I'm not at all certain, so hopefully someone else can confirm one way or other.
Or I could test my theory of course 🤦